spec/rewrite/address_spec.rb

```diff
@@ -0,0 +1,130 @@
+# frozen_string_literal: true
+
+require_relative '../spec_helper'
+
+describe EasyPost::Services::Address do
+  let(:client) { EasyPost::Client.new(ENV['EASYPOST_TEST_API_KEY']) }
```

Comment:
praise: nice work, this is the right way to do this so we can control it easily per file